docs/howto/functions.md

```diff
@@ -0,0 +1,34 @@
+(functions)=
+# Get SQL function results
+
+To output the result of a SQL function, such as `COUNT`, `MAX` or `SUM`, the
+result needs to be put in an output variable. The `AS` keyword can be used to
+select the result into an output variable. 
+
```

Comment:
I'd rewrite as:

To output the result of a SQL function such as `COUNT`, `MAX` or `SUM`, use the `AS` keyword to select the result into an output variable, then run the query.